aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -97,11 +97,12 @@ module aptos_framework::account {
     const EINVALID_ACCEPT_ROTATION_CAPABILITY: u64 = 10;
     ///
```

Comment:
This one is not needed as it's only for genesis, but might as well add a comment just in case. I forgot to in a commit some time ago. Can you add something along the line of:

/// Address to create is not a valid reserved address for Aptos framework